prime-angular-frontend/src/app/modules/enrolment/pages/health-authority/health-authority-form-state.ts

```diff
@@ -59,7 +59,7 @@ export class HealthAuthorityFormState extends AbstractFormState<HealthAuthority[
   }
 
   public buildForm(): void {
-    const healthAuthorities = this.configService.healthAuthorities
+    const healthAuthorities = this.configService?.healthAuthorities
```

Comment:
Feels like the `configService` should have this available since it is hardcoded into the mock.